UI/EmuScreen.cpp

```diff
@@ -263,6 +263,8 @@ void EmuScreen::bootGame(const Path &filename) {
 	if (!info || info->pending)
 		return;
 
+	SetExtraAssertInfo((info->id + " " + info->GetTitle()).c_str());
```

Comment:
It might be nice to clear it when exiting the game, so we don't get confused about UI asserts.

-[Unknown]

Reply:
Yeah, good point. Will do that.